app/models/bookshelf.rb

```diff
@@ -0,0 +1,15 @@
+require_dependency 'dpla/items'
+##
+# A model that represents a Book item retrieved from the search.
+#
+# For displaying the books in BookshelfController.
+# Search#conditions is overridden to forego the retrieval of facets.
+#
+# @see BookshelfController
+#
+class Bookshelf < Search
+
+  def conditions
+    { q: @term, facets: [], facet_size: 0 }.merge(@filters).merge(@args)
```

Comment:
I tested this locally, and it seemed to work the same for me if line 13 was simply:

`{ q: @term }.merge(@filters).merge(@args)`

Any reason to have explicit statements for `facets` and `facet_size`?